benches/benches/vm_initialization.rs

```diff
@@ -85,6 +87,9 @@ pub fn vm_initialization(c: &mut Criterion) {
         if size as u64 > consensus_params.script_params().max_script_data_length() {
             break
         }
+        if 2 * size as u64 > consensus_params.tx_params().max_size() {
```

Comment:
Can we put this into a helper with the name that explains why this should `break`? Same with the above `break`.

```
if `something_size_is_foo_bar_baz(size, consensus_params)` {
    break
}
```

It's not clear to me what the significance of `size` (size of what?) being half as big as the max tx size.